.github/workflows/update.yml

```diff
@@ -0,0 +1,152 @@
+name: Update vuln-list
+on: workflow_dispatch
+
+jobs:
+  update:
+    name: Update repo vuln-list
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Setup github user email and name
+      run: |
+        git config --global user.email "action@github.com"
+        git config --global user.name "GitHub Action"
+        
+    - name: Run NVD
+      env:
+        VULNLIST_REPOSITORY_OWNER: ${{ secrets.Owner }}
```

Comment:
If we don't specify it, `aquasecurity` will be used. Do we need this one?
https://github.com/aquasecurity/vuln-list-update/blob/11f2a4cbee7fe480dbfe65972784d61ac98aad62/main.go#L56

Reply:
No not needed, we can remove it, needed in my case to push to fork repo